hack/vcluster/partials/main.go

```diff
@@ -123,7 +45,44 @@ func main() {
 	if err != nil {
 		panic(fmt.Errorf("failed to parse schema JSON: %w", err))
 	}
+
+	for pair := schema.Properties.Oldest(); pair != nil; pair = pair.Next() {
+		walkTree(pair.Value, schema, pair.Key, "")
+	}
+
 	for _, path := range paths {
-		util.GenerateFromPath(schema, outputDir, path, defaults)
+		util.GenerateFromPath(schema, outputDir, strings.TrimPrefix(path, "/"), defaults)
+	}
+}
+
+func walkTree(node, parent *jsonschema.Schema, name, parentName string) bool {
+	if node == nil || getChildren(node, parent) == nil {
+		return true
+	} else {
+		parentName = fmt.Sprintf("%s/%s", parentName, name)
+		paths = append(paths, parentName)
+	}
+	children := getChildren(node, parent)
```

Comment:
Small nit, we should avoid else after return and exit early for readability, maybe something like:

```go
    if node == nil {
        return true
    }
    
    children := getChildren(node, parent)
    if children == nil {
        return true
    }
```